soc/arm/nordic_nrf/Kconfig.soc

```diff
@@ -3,4 +3,19 @@
 # Copyright (c) 2016-2018 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
+config NFCT_PINS_AS_GPIOS
```

Comment:
Perhaps it would be better to place it after the `source` command, so that this option appears in `menuconfig` after the MCU Selection one (and others from relevant `Kconfig.soc`).

Reply:
alright :)